filebeat/module/auditd/log/_meta/fields.yml

```diff
@@ -47,37 +35,137 @@
     - name: a0
       description: >
         The first argument to the system call.
+
+    - name: acct
+      type: alias
+      path: user.name
+      migration: true
+    - name: pid
+      type: alias
+      path: process.pid
+      migration: true
+    - name: ppid
+      type: alias
+      path: process.ppid
+      migration: true
     - name: res
-      description: >
-        The result of the system call (success or failure).
+      type: alias
+      path: event.outcome
+      migration: true
+    - name: record_type
+      type: alias
+      path: event.action
+      migration: true
     - name: geoip
       type: group
-      description: >
-        Contains GeoIP information gathered based on the `auditd.log.addr`
-        field. Only present if the GeoIP Elasticsearch plugin is available and
-        used.
       fields:
         - name: continent_name
-          type: keyword
-          description: >
-            The name of the continent.
-        - name: city_name
-          type: keyword
-          description: >
-            The name of the city.
-        - name: region_name
-          type: keyword
-          description: >
-            The name of the region.
+          type: alias
+          path: source.geo.continent_name
+          migration: true
         - name: country_iso_code
-          type: keyword
-          description: >
-            Country ISO code.
+          type: alias
+          path: source.geo.country_iso_code
+          migration: true
         - name: location
-          type: geo_point
-          description: >
-            The longitude and latitude.
+          type: alias
+          path: source.geo.location
+          migration: true
+        - name: region_name
+          type: alias
+          path: source.geo.region_name
+          migration: true
+        - name: city_name
+          type: alias
+          path: source.geo.city_name
+          migration: true
         - name: region_iso_code
-          type: keyword
-          description: >
-            Region ISO code.
+          type: alias
+          path: source.geo.region_iso_code
+          migration: true
+
+    # Fields below were not defined in 6.x but were still being populated.
+    - name: arch
+      type: alias
+      path: host.architecture
+      migration: true
+    - name: gid
+      type: alias
+      path: user.group.id
+      migration: true
+    - name: uid
+      type: alias
+      path: user.id
+      migration: true
+    - name: agid
+      type: alias
+      path: user.audit.group.id
```

Comment:
`user.group.id` was added to `fields.ecs.yml` in https://github.com/elastic/beats/pull/10275, but shouldn't fields like `user.audit.*`, `user.effective.*`, etc. also be added to a fields.yml file?

Reply:
Right, we'll do that separately. Only `user.group.*` is in ECS.

Perhaps we should add those additional fields only in Auditbeat and Filebeat for now?